packages/server/src/events/tasks.ts

```diff
@@ -24,7 +26,24 @@ export async function AddLikeActivity(email: string, articleId: number) {}
  * Task ( Runs after the article dislike is created successfully ).
  * Add dislike activity to the logged in user
  */
-export async function AddDislikeActivity(email: string, articleId: number) {}
```

Comment:
No, this function is for adding the dislike activity for the user. Not the like on the article itself. 

When any user likes/dislike any article , this function record the action to showcase on the action creator profile.